.prettierignore

```diff
@@ -1 +1,4 @@
-docs/**/*
\ No newline at end of file
+docs/**/*
+/index.js
+/index-with-locales.js
```

Comment:
Prevent converting `var` to `const`. Files `index.js` `index-with-locales.js` need to be es5